doc/sources/samples.rst

```diff
@@ -53,13 +53,21 @@ If you want to run them locally, refer to `these instructions
    Kmeans for spoken arabic digit dataset <samples/kmeans.ipynb>
    DBSCAN for spoken arabic digit dataset <samples/dbscan.ipynb>
 
+.. rubric:: Dimensionality Reduction Tasks
+
+.. toctree::
+   :maxdepth: 1
+
+   Intel® Extension for Scikit-learn TSNE example <samples/tsne.ipynb>
+
 .. rubric:: Jupyter Notebook Samples
 
 .. toctree::
    :maxdepth: 1
 
    Intel® Extension for Scikit-Learn* Getting Started Sample <https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Getting-Started-Samples/Intel_Extension_For_SKLearn_GettingStarted>
    Intel® Extension for Scikit-Learn*: SVC for Adult dataset Performance Sample <https://github.com/oneapi-src/oneAPI-samples/tree/master/AI-and-Analytics/Features-and-Functionality/Intel_Extension_For_SKLearn_Performance_SVC_Adult>
+   Utilizing daal4py in Data Science Workflows <samples/daal4py_data_science.ipynb>
```

Comment:
We are moving towards sklearnex estimators interface. We are not going to extend the legacy daal4py API.
I think it's better not to extend the docs with the additional daal4py samples.

It would be better to update the notebook using sklearnex API and then add the link to it.

Reply:
Then I guess removing those .ipynb files might be a better option.

Reply:
Left that `daal4py_data_science.ipynb` out from the docs and cleared the warning instead by not copying to the samples folder.